libconfig/src/config_content_node.c

```diff
@@ -187,13 +187,14 @@ const char * config_content_node_iget_as_relpath( config_content_node_type * nod
 }
 
 const char * config_content_node_iget_as_executable( config_content_node_type * node, int index ) {
-  config_schema_item_assure_type(node->schema, index,
-        CONFIG_PATH + CONFIG_EXISTING_PATH + CONFIG_EXECUTABLE );
+  config_schema_item_assure_type(node->schema, index, CONFIG_EXECUTABLE );
   {
     const char * config_value = config_content_node_iget(node , index);
 
     char* path_value = NULL;
-    if( !util_file_exists( config_value ) )
+    if( !strstr( config_value, UTIL_PATH_SEP_STRING )
+     && !strstr( config_value, "/" )
```

Comment:
`UTIL_PATH_SEP_STRING` is set depending on your system, so is the second condition necessary?